server/README.md

```diff
@@ -473,9 +116,12 @@ See more about available logging and log levels at https://docs.rs/env_logger/la
 ### Missing metrics in upstream server
 
 #### Possible reasons
-- Old edge version. In order to guarantee metrics on newer Unleash versions, you will need to be using Edge v17.0.0 or newer
-- Old SDK clients. We've seen some clients, particularly early Python (1.x branch) as well as earlier .NET SDKs (we recommend you use 4.1.5 or newer) struggle to post metrics with the strict headers Edge requires.
+
+- Old edge version. In order to guarantee metrics on newer Unleash versions, you will need to be using Edge v17.0.0 or
+  newer
+- Old SDK clients. We've seen some clients, particularly early Python (1.x branch) as well as earlier .NET SDKs (we
+  recommend you use 4.1.5 or newer) struggle to post metrics with the strict headers Edge requires.
 
 ## Development
 
-See our [Contributors guide](./CONTRIBUTING.md) as well as our [development-guide](./development-guide.md)
+See our [Contributors guide](../CONTRIBUTING.md) as well as our [development-guide](../development-guide.md)
```

Comment:
Just curious: Is `server/README.md` a duplicate of the root `README.md`?  Why?

Reply:
So, due to how we setup the workspace, it's server/README.md that gets published to crates.io, but README.md that is available on github/unleash docs.